shared/test/middleware/files_api/test_public_thumbnails.rb

```diff
@@ -10,62 +10,6 @@ def setup
     Aws::S3::Client.expects(:new).never
   end
 
-  def test_adult_thumbnail
```

Comment:
In this situation it may be preferable to use [`skip`](https://apidock.com/ruby/MiniTest/Assertions/skip).

```rb
def test_adult_thumbnail
  skip 'Disabled while we gather Azure moderation stats. (PR 24626)'
  # ...rest of test...
end
```

Reply:
Good idea! I always forget that I can do that. Thanks.